jwala-integration-test/src/test/postman/jwala-collection.postman_collection.json

```diff
@@ -7182,6 +7182,128 @@
 						"description": ""
 					},
 					"response": []
+				}
+			]
+		},
+		{
+			"name": "u) health check application test",
+			"description": "",
+			"item": [
+				{
+					"name": "start jvm with wait true",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "179a918a-9d12-4d3a-9594-17818fefa228",
+								"type": "text/javascript",
+								"exec": [
+									"setTimeout(function(){}, 5000);",
+									"tests[\"Status code is 200\"] = responseCode.code === 200;"
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"controlOperation\":\"start\"}"
+						},
+						"url": {
+							"raw": "{{url}}/v1.0/jvms/1/commands?wait=TRUE",
```

Comment:
We can't assume that there will always be a JVM ID of 1 available. This needs to be a an environment variable that references a JVM created as part of the postman tests.